src/mailroom.py

```diff
@@ -0,0 +1,339 @@
+# -*- coding: utf-8 -*-
+"""
+    TODO: sort user report before printing
+    TODO: create function to create a new user
+
+"""
+import os
+import sys
+from collections import OrderedDict
+
+
+def initialize_donor_dict():
+    '''function to create the dictionary of dictionary to
+       hold donor information
+    '''
+    John_Doe = {'name': 'John Doe',
+                'total_donation': 0,
+                'num_donations': 0,
+                'avg_donation': 0,
+                'last_donation': 0}
+
+    Neil_Armstrong = {'name': 'Neil Armstrong',
+                      'total_donation': 23000000,
+                      'num_donations': 4,
+                      'avg_donation': 5750000,
+                      'last_donation': 10000000}
+
+    Sara_Smith = {'name': 'Sara Smith',
+                  'total_donation': 100,
+                  'num_donations': 2,
+                  'avg_donation': 50,
+                  'last_donation': 35}
+
+    Ralf_the_Dog = {'name': 'Ralf the Dog',
+                    'total_donation': 1000,
+                    'num_donations': 1,
+                    'avg_donation': 1000,
+                    'last_donation': 1000}
+
+    donor_dict = {'John Doe': John_Doe,
+                  'Neil Armstrong': Neil_Armstrong,
+                  'Sara Smith': Sara_Smith,
+                  'Ralf the Dog': Ralf_the_Dog,
+                  }
```

Comment:
This works, but it seems a bit wordy in terms of what you are storing.  What if your dictionary looked more like this:
```python
DONORS = {
    'Niel Armstrong': [2000, 100000, 100000],
    'Sara Smith': [50, 40, 10],
}
```

Is there any information in your storage above that could not be calculated on the fly from a structure like this?

Even better, with this new approach, you can simply append a new donation to the list of donations a donor has made over time.  Much easier than what you can do with the structure above.